.chloggen/attributes-routing-key.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
I approved, and reconsidered - we should skip changelog for a docs-only change like this.

Reply:
sounds good to me! removed.